src/finch/autoschedule/optimize.py

```diff
@@ -58,12 +62,14 @@ def rule_0(node):
                 name = Alias(gensym("A"))
                 return Subquery(name, tbl)
 
-    return PostWalk(rule_0)(root)
+    ret = PostWalk(rule_0)(root)
+    assert ret is not None
+    return ret
 
 
 def pretty_labels(root: LogicNode) -> LogicNode:
-    fields = {}
-    aliases = {}
+    fields: dict[Field, Field] = {}
+    aliases: dict[Alias, LogicNode] = {}
```

Comment:
I think it can also be `dict[Alias, Alias]`

Reply:
Done in the [force-pushed commit](https://github.com/finch-tensor/finch-tensor-lite/compare/01aeb2572f66e5e5beae51b12119d49006f44342..3baabd6a6d5be47927341235037bbfc1ae39772a).